src/components/Firestore/index.tsx

```diff
@@ -54,7 +56,24 @@ export const Firestore: React.FC<Props> = ({ config, projectId }) => {
   }
 
   async function handleClearData(api: DatabaseApi) {
-    const shouldNuke = await promptClearAll();
+    const shouldNuke = await confirm({
```

Comment:
This dialogs change got reverted, please undo (use the external file)

Reply:
Good catch, reverted